book/02-using-atom/sections/02-text.asc

```diff
@@ -289,6 +289,8 @@ console.log("crash");
 
 The string `"crash"` would be initially selected and pressing tab again would place the cursor after the `;`
 
+WARNING: Snippet keys, unlike CSS selectors, can only be repeated once per level. If there are duplicate keys, then only the last one will be read. See <<_cson,Configuring with CSON>> for more information.
```

Comment:
> If there are duplicate keys, ...

Perhaps "If there are duplicate keys **at the same level**..." just to be :gem: clear?